source/common/network/dns_impl.h

```diff
@@ -91,5 +92,5 @@ class DnsResolverImpl : public DnsResolver {
   std::unordered_map<int, Event::FileEventPtr> events_;
 };
 
-} // Network
-} // Envoy
+} // namespace Network
```

Comment:
OK, I ran the format tests with clang-format from clang 5.0, and it fails to pass the check without these changes. I must conclude that clang-format-3.6 will not require these changes.